### PR TITLE
Fixed inaccuracies in where merchants can be registered to use GoCardless

### PR DESCRIPTION
**Pull request by octaveauger**
The previous text mentioned that you could use GC even if you were not registered in the UK, which is only true for Pro.

### CODE REVIEW

**Comment by samjewell** [comment]
@octaveauger do you want me to review, and then merge/deploy?

**Comment by octaveauger** [comment]
@samjewell yes please - sorry, forgot to mention anyone in my initial pull request!

File: app/pages/faq/merchants/index.html [hunk 1]
```diff
@@ -457,12 +457,12 @@ <h2 class="u-text-heading-light u-text-m u-color-heading">International payments
...
               Can only UK businesses use GoCardless?
             </h3>
-            <p class="para">No. You just need to have a GBP bank account so we can pay you. You do not need to be a UK registered business or individual.</p>
+            <p class="para">Yes for the time being, unless you use GoCardless Pro.</p>
 
             <h3 class="section-heading u-text-heading-light u-color-heading u-margin-Vm u-text-s">
               When can I collect from Europe?
             </h3>
-            <p class="para">With GoCardless, any merchant registered in the U.K. or Ireland can collect from the Single Europe Payments Area. You will need to have a Euro account open in the UK or a Eurozone country. <br><br>If you take more than 500 payments a month and want to use our Pro solution, you can be registered anywhere in the world.</p>
+            <p class="para">With GoCardless, any merchant registered in the U.K. can collect from the Single European Payments Area. You will need to have a Euro account open in the UK or a Eurozone country. <br><br>If you take more than 500 payments a month and want to use our Pro solution, you can be registered anywhere in the world.</p>
```
Comment (samjewell) [inline app/pages/faq/merchants/index.html]: Looks great man. While we're here, I think we should ditch the <br>'s, and start a new paragraph here.

I'm happy to do this and push to your branch, if that's cool?

**Comment by samjewell** [comment]
Oh also, if you know what we should say instead of this, do you fancy updating it?
`GoCardless will not be available outside Europe in 2014...`
https://github.com/gocardless/splash-pages/pull/397/files?diff=unified#diff-75a0bc0b773ee49dd59b8e90d136e66bR492

**Comment by octaveauger** [comment]
@samjewell I have updated that sentence to remove the 2014 bit but stayed non-committing in terms of where we could expand.

**Comment by samjewell** [comment]
@octaveauger I can't see your changes to the 2014 thing - did you push them yet?

**Comment by octaveauger** [comment]
@samjewell my bad, the error message in my push wasn't very noticeable - now pushed properly after merging your changes. See https://github.com/gocardless/splash-pages/commit/36340cc728613bd78f27d269382dcdc81a4e60b5 for my changes.

**Comment by samjewell** [comment]
@octaveauger AWESOME!

**Pull request merged**